src/marks/Link.js

```diff
@@ -60,6 +60,8 @@ const Link = TipTapLink.extend({
 		...mark.attrs,
 		href: domHref(mark),
 		rel: 'noopener noreferrer nofollow',
+		// Disable default browser opening handler (only relevant in readonly mode)
+		onclick: "return false;"
```

Comment:
Is this still needed with the clickPreventer plugin? I'm wondering if this might be an issue with our CSP if that is considered inline javascript

Reply:
ah, thanks. fixed it